bazel/repository_locations.bzl

```diff
@@ -67,10 +67,10 @@ DEPENDENCY_REPOSITORIES = dict(
         use_category = ["build"],
     ),
     envoy_build_tools = dict(
-        sha256 = "9d348f92ae8fb2495393109aac28aea314ad1fb013cdec1ab7b1224f804be1b7",
-        strip_prefix = "envoy-build-tools-823c2e9386eee5117f7ef9e3d7c90e784cd0d047",
-        # 2020-04-07
-        urls = ["https://github.com/envoyproxy/envoy-build-tools/archive/823c2e9386eee5117f7ef9e3d7c90e784cd0d047.tar.gz"],
+        sha256 = "",
```

Comment:
I unfortunately missed this in review but isn't having the sha256 field empty bypassing the check? A prior example of this creeping into the code base is libprotobuf_mutator https://github.com/envoyproxy/envoy/blob/master/bazel/repository_locations.bzl#L166

I can fix in a PR but is it possible to have a check to prevent empty sha256 strings? /cc @htuch @lizan

Reply:
@moderation good idea, see https://github.com/envoyproxy/envoy/pull/11154